scss/components/textarea.scss

```diff
@@ -0,0 +1,19 @@
+@import "./../settings";
+@import "./../mixins";
+@import "./../functions";
+/*!
+.fd-textarea
+*/
+$block: #{$fd-namespace}-textarea;
+.#{$block} {
+    @include fd-form-text();
+    //could not get this to work with variables - they are stored in core/root?
+    height: 72px;
+    width: 100%;
+    padding-top: $fd-forms-padding;
+    &--compact {
+        //could not get this to work with variables - they are stored in core/root?
+        height: 56px;
```

Comment:
The variables that are storing the height calculations for textarea and textarea--compact are stored in `core/root`. I didn't want to import that entire file in here for 2 variables from it. Thoughts?